test/uk/gov/hmrc/gform/sharedmodel/ExampleData.scala

```diff
@@ -192,7 +192,7 @@ trait ExampleFormTemplate { dependsOn: ExampleAuthConfig with ExampleSection wit
     submitSuccessUrl = submtSuccessUrl,
     submitErrorUrl = submitErrorUrl,
     sections = allSections,
-    List.empty[AckSection],
+    acknowledgementSection = AcknowledgementSection("", None, None, Nil),
```

Comment:
It would be great if example data contained exemplary `AcknowledgemendSection` not just mock plumbing.